Userland/Libraries/LibWeb/HTML/HTMLInputElement.cpp

```diff
@@ -1443,6 +1459,38 @@ void HTMLInputElement::apply_presentational_hints(CSS::StyleProperties& style) c
     });
 }
 
+// https://html.spec.whatwg.org/multipage/input.html#attr-input-maxlength
```

Comment:
Link should be https://html.spec.whatwg.org/multipage/input.html#dom-input-maxlength
which is what actually contains the spec text copied below